backend/src/election/api.rs

```diff
@@ -278,6 +278,9 @@ pub struct ElectionAndCandidatesDefinitionImportRequest {
     #[schema(nullable = false)]
     #[serde(skip_serializing_if = "Option::is_none")]
     number_of_voters: Option<u32>,
+    #[schema(nullable = false)]
+    #[serde(skip_serializing_if = "Option::is_none")]
+    file_name: Option<String>,
```

Comment:
I would change this field to polling_station_file_name just to be sure not to confuse it with any of the other files uploaded.